packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -335,27 +344,27 @@ const demoProjectAnswers = () => {
 }
 
 const helloWorldPrompts = () => {
-    const validProjectId = (s) =>
-        /^[a-z0-9-]{1,20}$/.test(s) ||
-        'Value can only contain lowercase letters, numbers, and hyphens.'
+    const validProjectName = (s) =>
+        /^[a-zA-Z0-9-\s]{1,20}$/.test(s) ||
+        'Value can only contain letters, numbers, space and hyphens.'
     const questions = [
         {
-            name: 'projectId',
-            validate: validProjectId,
-            message: 'What is your project ID (example-project) in Managed Runtime Admin?'
+            name: 'projectName',
```

Comment:
Just going to start by recapping what was discussed in the Feb 2 tech meeting.

Right now, we don't actually use the project name for anything. The entry that is used as the resulting directory name and the entry that is set in the package.json 'name' property is the project id (aka. the slug). For the project to be deployable in MRT, this slug needs to match the slug on MRT.

Decoupling the MRT slug from the package.json name is a separate task. But for now, it seems we are fine with asking for a `name` rather than an `id` as regardless of which we ask for, the user will need to do additional config work will be required before they can push the project onto MRT.

Reply:
Switching gears, could we use the project name the user inputs as the output dir if the --outputDir arg isn't set?

Reply:
Thanks, Vicent, I missed that part. 

The generator now uses the Project name slugfied as the name of the output directory for the `retail-react-app` and  `hello-world` presets.